plan/physical.go

```diff
@@ -90,8 +98,9 @@ func validatePhysicalPlan(plan *PlanSpec) error {
 
 type physicalPlanner struct {
 	*heuristicPlanner
-	defaultMemoryLimit int64
-	disableValidation  bool
+	defaultMemoryLimit     int64
+	disableValidation      bool
+	disableIntegrityChecks bool
```

Comment:
Can `disableIntegrityChecks` be the same option as `disableValidation`?  The only reason to disable the checks is enable testing.